scripting/get5.sp

```diff
@@ -345,6 +347,12 @@ public void OnPluginStart() {
                    "Seconds to countdown before veto process commences. Set to \"0\" to disable.");
   g_WarmupCfgCvar =
       CreateConVar("get5_warmup_cfg", "get5/warmup.cfg", "Config file to exec in warmup periods");
+  g_RoundBackupFormatCvar = CreateConVar(
+      "get5_backup_format", "get5_backup_match%s_map%d_round%d.cfg",
+      "Path where round backups will be stored.");
+  g_PreLiveBackupFormatCvar = CreateConVar(
+      "get5_backup_format", "get5_backup_match%s_map%d_prelive.cfg",
```

Comment:
I don't think we can have two convars with the same name. Both are `get5_backup_format` here.